compiler/qsc_passes/src/capabilitiesck.rs

```diff
@@ -176,6 +178,7 @@ impl<'a> Visitor<'a> for Checker<'a> {
         // We only care about callables.
         if let ItemKind::Callable(callable_decl) = &item.kind {
             self.set_current_callable(item.id);
+            self.check_measurement(&item.attrs, item.span);
```

Comment:
Rather than underline the whole item declaration, I think just underlining the name would be less overwhelming (and less likely to hide other errors). So something like:

```suggestion
            self.check_measurement(&item.attrs, callable_decl.name.span);
```